force-app/main/default/classes/SOQL.cls

```diff
@@ -85,9 +103,20 @@ public with sharing class SOQL {
      * @return A map of Task Ids to Task records.
      * Update the corresponding test class and testGetTasksDueNextWeek method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static Map<Id,Task> getTasksDueNextWeek() {
+        Date today =  Date.today();
+        Date nextWeek = today.addDays(7);
```

Comment:
Try to use the Next_N_Days:7 date literal  https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_select_dateformats.htm